File: back-end/golang/1-simple-zapier-trigger/trigger.go
package main

import (
	"fmt"
	"io/ioutil"
	"log"
	"net/http"
)

func main() {
	const ZapierWebhookURL = "https://hooks.zapier.com/hooks/catch/YOUR-ZAPIER-WEBHOOK/oxZ7q6b3/"

	res, err := http.Get(ZapierWebhookURL)
	if err != nil {
		log.Println("Zapier webhook request failed.")
		log.Fatalln(err)
	}

	data, err := ioutil.ReadAll(res.Body)
	res.Body.Close()

	if err != nil {
		log.Println("Zapier webhook request failed.")
		log.Fatalln(err)
	}

	// You can use the response data via:
	fmt.Printf("%s", data)

	log.Println("Zapier webhook request was successful.")
}

File: back-end/python3/1-simple-zapier-trigger/trigger.py
import urllib

zapier_webhook_url = 'https://hooks.zapier.com/hooks/catch/YOUR-ZAPIER-WEBHOOK/oxZ7q6b3/'

with urllib.request.urlopen(zapier_webhook_url) as response:
    response_code = response.getcode()

    # You can read response data with:
    # data = response.read()

    if response_code == 200:
        print("Zapier webhook request was successful")
    else:
        print("Zapier webhook request failed")
